docs/publish/pom.xml

```diff
@@ -154,6 +154,28 @@
                             </artifactItems>
                         </configuration>
                     </execution>
+                    <!--
+                        Copy the hello.war to the docs directory.
+                    -->
+                    <execution>
+                        <id>hello-war</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>org.glassfish.main.docs</groupId>
+                                    <artifactId>EclipseGlassFishHelloSample</artifactId>
```

Comment:
This probably needs to be updated.

Reply:
Yes, that's correct. After renaming the project and the artifact, this should be also replaced with `sample-app`, @samsonsouley

Reply:
Thanks. it is done